edb/lib/ext/ai.edgeql

```diff
@@ -316,57 +316,63 @@ CREATE EXTENSION PACKAGE ai VERSION '1.0' {
             ext::ai::text_gen_model_context_window := "128000";
     };
 
-    # Mistral models
-    create abstract type ext::ai::MistralEmbedModel
-        extending ext::ai::EmbeddingModel
+    create abstract type ext::ai::OpenAI_O1_PreviewModel
+        extending ext::ai::TextGenerationModel
     {
         alter annotation
-            ext::ai::model_name := "mistral-embed";
+            ext::ai::model_name := "o1-preview";
         alter annotation
-            ext::ai::model_provider := "builtin::mistral";
+            ext::ai::model_provider := "builtin::openai";
         alter annotation
-            ext::ai::embedding_model_max_input_tokens := "8192";
+            ext::ai::text_gen_model_context_window := "128000";
+    };
+
+    create abstract type ext::ai::OpenAI_O1_MiniModel
+        extending ext::ai::TextGenerationModel
+    {
         alter annotation
-            ext::ai::embedding_model_max_batch_tokens := "16384";
+            ext::ai::model_name := "o1-mini";
         alter annotation
-            ext::ai::embedding_model_max_output_dimensions := "1024";
+            ext::ai::model_provider := "builtin::openai";
+        alter annotation
+            ext::ai::text_gen_model_context_window := "128000";
     };
 
-    create abstract type ext::ai::MistralSmallModel
+    # Mistral models
+    create abstract type ext::ai::MistralLargeModel
         extending ext::ai::TextGenerationModel
```

Comment:
These are breaking changes. What's our policy on breaking `ext::ai`?

Reply:
oh what is a breaking change? I didn' t realize, I just wanted to update models

Reply:
For future reference: we very strongly strongly prefer not to add breaking changes into ext modules. Because this makes database dumps un-restorable on newer versions of EdgeDB.

Reply:
I believe I fixed this : ) Thank u a lot!